packages/scripts/src/upgrade-tests/voting2/1_Propose.ts

```diff
@@ -96,9 +96,10 @@ async function main() {
 
   let votingUpgrader;
 
-  if (process.env[TEST_DOWNGRADE])
+  if (process.env[TEST_DOWNGRADE]) {
+    if (!process.env[EMERGENCY_EXECUTOR]) throw new Error("Must provide EMERGENCY_EXECUTOR");
     votingUpgrader = await deployVotingUpgraderAndRunDowngradeOptionalTx(
-      (await hre.ethers.getSigners())[0].address,
+      String(process.env[EMERGENCY_EXECUTOR]),
```

Comment:
nit: is this just to appease typescript because it always produces a string (it will stringify undefined as "undefined")?

I might suggest doing something like this instead:
```ts
const emergencyExecutor = process.env[EMERGENCY_EXECUTOR];
if (!emergencyExecutor) throw new Error("Must provide EMERGENCY_EXECUTOR");
votingUpgrader = await deployVotingUpgraderAndRunDowngradeOptionalTx(emergencyExecutor, ... );
```

By extracting the value before checking, ts can understand that you already ruled out the undefined case with the error check above.